Documentation/CRDs/.pages

```diff
@@ -0,0 +1,3 @@
+nav:
+    - ceph-cluster-crd.md
```

Comment:
How about we order the CRD pages with folders? Something like this:
- Cluster CRD
- Block
   - Block Pool CRD
   - Block Pool RADOS Namespace CRD
   - RBD Mirror CRD
- Shared Filesystem
   - Filesystem CRD
   - Filesystem Mirror CRD
   - NFS Server CRD
- Object Store
   - Object Store CRD
   - Object Multisite CRDs
   - Object Store User CRD
- Client CRD

Reply:
I have renamed the `Block` section to `Block Storage` to be better recognizable coming from the `Storage Configuration` section.